modules/luci-base/htdocs/luci-static/resources/validation.js

```diff
@@ -563,7 +563,7 @@ var ValidatorFactory = baseclass.extend({
 		},
 
 		timehhmmss: function() {
-			return this.assert(this.value.match(/^[0-6][0-9]:[0-6][0-9]:[0-6][0-9]$/),
+			return this.assert(this.value.match(/^(?:[01]\d|2[0-3]):[0-5]\d:[0-5]\d$/),
```

Comment:
This should give us what we need
```js
			return this.assert(this.value.match(^(?:[01]\d|2[0-4]):[0-5]\d:(?:[0-5]\d|60)$),
```

Reply:
Confirm ;-)

Reply:
I've updated the regex with the suggestion, thanks!